src/test/scala/akka/persistence/redis/RedisUtilsSpec.scala

```diff
@@ -0,0 +1,42 @@
+package akka.persistence.redis
+
+import com.typesafe.config.ConfigFactory
+import org.scalatest.{ FlatSpec, Matchers }
+
+class RedisUtilsSpec extends FlatSpec with Matchers {
+
+  "A RedisClient" should "use redis.sentinels as config if sentinel-list is also present" in {
+    val config = ConfigFactory.parseString(s"""
+         |redis {
+         |  mode = sentinel
+         |  master = foo
+         |  database = 0
+         |  sentinel-list = "host1:1234,host2:1235"
+         |  sentinels = [
+         |   {
+         |     host = "host3"
+         |     port = 1236
+         |    },
+         |    {
+         |      host = "host4"
+         |      port = 1237
+         |     }
+         |   ]
+         |}""".stripMargin)
+    val sentinels = RedisUtils.sentinels(config)
+    assert(sentinels.contains(List(("host3", 1236), ("host4", 1237))))
```

Comment:
I would prefer using the `should be` matcher. More than containing these elements, the sentinels must be exactly this list.

Please use

```scala
sentinels should be(List(("host3", 1236), ("host4", 1237)))
```